From 9a5b39a2e6c6f720064399c9fd1ee4365050180a Mon Sep 17 00:00:00 2001
From: clovemint <shaleenjha10@gmail.com>
Date: Wed, 13 May 2020 09:12:53 +0000
Subject: [PATCH] Create Patch File

---
 client3.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client3.py b/client3.py
index f1771c3..2a14815 100644
--- a/client3.py
+++ b/client3.py
@@ -10,7 +10,7 @@
 #  The above copyright notice and this permission notice shall be included in
 #  all copies or substantial portions of the Software.
 #
-#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WAR:RANTY    OF ANY KIND, EXPRESS
 #  OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 #  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 #  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-- 
2.17.1

